qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/SubscriptionPage.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+
+public class SubscriptionPage extends Page {
+    
+    private static final By RELEASE_DICTIONARY_URL_FIELD = By.id("subscription-url");
+    private static final By TOKEN_URL_FIELD = By.id("subscription-token");
+    private static final By SAVE_CHANGES_BUTTON = By.cssSelector("#body-wrapper > subscription > div:nth-child(1) > form > fieldset > div:nth-child(4) > button:nth-child(1)");
+    private static final By CANCEL_CHANGES_BUTTON = By.cssSelector("#body-wrapper > subscription > div:nth-child(1) > form > fieldset > div:nth-child(4) > button:nth-child(2)");
+    private static final By UNSUBSCRIBE_BUTTON = By.cssSelector("#body-wrapper subscription div:nth-child(2) form fieldset p:nth-child(3) button");
```

Comment:
it would be a good idea to use better element locators than retrieving elements through div:nth-child(x) values. These can easily break when UI changes comes.